packages/react-component-library/src/components/NumberInput/NumberInput.test.tsx

```diff
@@ -10,14 +10,23 @@ describe('NumberInput', () => {
   let onChangeSpy: (event: any) => void
 
   function assertInputValue(expected: string) {
-    it('should set the new value', () => {
+    it('should set the new value attribute', () => {
       const input = wrapper.getByTestId(
         'number-input-input'
       ) as HTMLInputElement
       expect(input.value).toEqual(expected)
     })
   }
 
+  function assertAriaValue(expected: number) {
```

Comment:
I like what you have done here. I am wondering if the assertion in here should just be part of `assertInputValue`. Then it is done every time we are asserting the value. Do you have thoughts? Either that or we `assertAriaValue` for every `assertInputValue` as it could become confusing when maintaining these tests. When to use `assertInputValue` and when to use `assertAriaValue`.

Reply:
I kept them separate because they are actually asserting different values - the aria value expects a number and the the component value expects a string that sometimes also includes a unit as a prefix/suffix.